packages/shared/lib/contexts/governance/actions/registerProposalsFromPrimaryNode.ts

```diff
@@ -0,0 +1,19 @@
+import { get } from 'svelte/store'
+
+import { activeProfile } from '@core/profile/stores'
+
+import { registerProposalsFromNode } from './registerProposalsFromNode'
+
+export async function registerProposalsFromPrimaryNode(): Promise<void> {
```

Comment:
If we don't have a primary node we do not need to throw an error, just dont attempt to register any proposals for that node.